lib/Content/View/Filter/ContentCreateViewFilter.php

```diff
@@ -105,17 +115,19 @@ private function resolveContentCreateData(
      * @param \EzSystems\RepositoryForms\Data\Content\ContentCreateData $contentCreateData
      * @param string $languageCode
      *
-     * @return \Symfony\Component\Form\FormInterface
+     * @param array $preferredLanguages
```

Comment:
Probably should be

```suggestion
     * @param string[] $preferredLanguages
```

 if i'm wrong, feel free to ignore this comment